app/src/main/res/layout/list_tagged_organizer_item.xml

```diff
@@ -16,55 +16,47 @@
   ~ limitations under the License.
   -->
 
-
-<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:card_view="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:layout_marginLeft="8dip"
-  android:layout_marginRight="8dip"
-  android:layout_marginTop="8dip"
-  card_view:cardCornerRadius="5dp">
-
-  <LinearLayout
+<LinearLayout
     android:id="@+id/contentContainer"
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Fixing that! Is there a way for the indentation to be added to the checkstyle to prevent this from happening again? (I mean for future contributors_